### ISSUE TITLE
Undefined variable $path

### ISSUE THREAD

**Issue by eaili** [clemdesign/grav-plugin-social-seo-metatags#19]
Hi,
I get the critical error "Undefined variable: path" on Grav 1.7.7 and the plugin version 2.1.0 .
The stack-trace is

> /var/www/html/developers-t6/system/src/Grav/Common/Debugger.php(843): Whoops\Run->handleError() 
> /var/www/html/developers-t6/user/plugins/social-seo-metatags/social-seo-metatags.php(281): Grav\Common\Debugger->deprecatedErrorHandler() 
> /var/www/html/developers-t6/user/plugins/social-seo-metatags/social-seo-metatags.php(313): Grav\Plugin\SocialSEOMetaTagsPlugin->getDefaultImage() 
> /var/www/html/developers-t6/user/plugins/social-seo-metatags/social-seo-metatags.php(104): Grav\Plugin\SocialSEOMetaTagsPlugin->getTwitterCardMetatags() 
> /var/www/html/developers-t6/vendor/symfony/event-dispatcher/EventDispatcher.php(264): Grav\Plugin\SocialSEOMetaTagsPlugin->onPageInitialized() 
> /var/www/html/developers-t6/vendor/symfony/event-dispatcher/EventDispatcher.php(239): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() 
> /var/www/html/developers-t6/vendor/symfony/event-dispatcher/EventDispatcher.php(73): Symfony\Component\EventDispatcher\EventDispatcher->callListeners() 
> /var/www/html/developers-t6/system/src/Grav/Common/Grav.php(550): Symfony\Component\EventDispatcher\EventDispatcher->dispatch() 
> /var/www/html/developers-t6/system/src/Grav/Common/Processors/PagesProcessor.php(44): Grav\Common\Grav->fireEvent() 
> /var/www/html/developers-t6/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\PagesProcessor->process() 

It seems like adding `$path = null` in the beginning of getDefaultImage() solves it, but feels like it should have other side-effects as well.
Do you have any idea of what's going on? Thanks in advance!

**Comment by clemdesign** [comment]
Hi,

You're right. There is an issue in getDefaultImage(). I correct it and propose un bugfix.

**Comment by clemdesign** [comment]
Inside `getDefaultImage()`, I add the following fix:

```

    if (isset($path) && is_file($path)) {
      $image = MediumFactory::fromFile($path);
    }
```

**Comment by clemdesign** [comment]
Fix available in release 2.1.1 (https://github.com/clemdesign/grav-plugin-social-seo-metatags/releases/tag/2.1.1)
